apps/search/test/marionette/lib/search.js

```diff
@@ -124,6 +126,12 @@ Search.prototype = {
     this.client.helper.waitForElement('body');
   },
 
+  searchDataVersion: function() {
+    var file = path.resolve(__dirname +
```

Comment:
Since we already have two places we have to bump, might as well hardcode the version here so we can add a comment?